v4/source/system.yaml

```diff
@@ -31,6 +31,16 @@
             $ref: "#/definitions/StatusOK"
         '403':
           $ref: '#/responses/Forbidden'
+      x-code-samples:
+        - lang: 'Go'
+          source: |
+            import "github.com/mattermost/platform/model"
+
+            Client := model.NewAPIv4Client("https://your-mattermost-url.com")
+            Client.Login("email@domain.com", "Password1")
+
+            // DatabaseRecycle
+            flag, err := Client.DatabaseRecycle()
```

Comment:
Also maybe remove the `// DatabaseRecycle` comments, not sure if they're much help since it's just the name of the function